src/Agoda.Analyzers/AgodaCustom/AG0026EnsureOnlyCssSelectorIsUsedToFindElements.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.CodeAnalysis;
+using Agoda.Analyzers.Helpers;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Text.RegularExpressions;
+
+namespace Agoda.Analyzers.AgodaCustom
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class AG0026EnsureOnlyCssSelectorIsUsedToFindElements : InvocationExpressionAnalyzer
+    {
+        public const string DIAGNOSTIC_ID = "AG0026";
+
+        private static readonly LocalizableResourceString _msg = new LocalizableResourceString(
+                nameof(CustomRulesResources.AG0026Title), CustomRulesResources.ResourceManager, typeof(CustomRulesResources));
+              
+        protected override DiagnosticDescriptor Descriptor => new DiagnosticDescriptor(
+                DIAGNOSTIC_ID,
+                _msg,
+                _msg,
+                AnalyzerCategory.CustomQualityRules,
+                DiagnosticSeverity.Warning,
+                AnalyzerConstants.EnabledByDefault,
+                DescriptionContentLoader.GetAnalyzerDescription(nameof(AG0026EnsureOnlyCssSelectorIsUsedToFindElements)),
+                "https://github.agodadev.io/pages/standards-c-sharp/code-standards/gui-testing/css-selectors.html",
+                WellKnownDiagnosticTags.EditAndContinue
+            );
+
+        protected override string NamespaceAndType => "OpenQA.Selenium.By";
+        protected override Regex Regex => new Regex("^((?!CssSelector).)*$");
```

Comment:
Could this be named more clearly? Maybe `MatchForbiddenMethodName`?

And I can't work out what the `.` in the regex is doing - is this a literal dot (so should be `\.`), or is it meant to match any character? Either way, I'm confused... though it obviously seems to work.